cortex-plugins-core.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "cells-rails", "~> 0.0.6"
   s.add_dependency "cells-haml", "~> 0.0.10"
   s.add_dependency "mimemagic", "~> 0.3.2"
-  s.add_dependency "ckeditor", "~> 4.2.0"
+  s.add_dependency "ckeditor", "= 4.2.0"
```

Comment:
This is covered in the 0.4.8 version of the gem - you don't need to add this. It was hot fixed to master so I can do a retroactive application for develop. But you can nix this change